src/utils/api/helper.ts

```diff
@@ -1360,3 +1408,22 @@ export const evaluateFeatureFlag = async (
     )
   )();
 };
+
+export const checkLogout = (onLogoutCallBack: () => void) => {
```

Comment:
```suggestion
export const checkLogout = (onLogoutCallBack: typeof constVoid) => {
  pipe(
    SessionItems.authToken,
    O.fromNullable,
    O.fold(constVoid, async () => {
      await logoutUser({
        onError: onLogoutCallBack,
        onResponse: onLogoutCallBack,
      });
    })
  );
};```

with constVoid -> https://gcanti.github.io/fp-ts/modules/function.ts.html#constvoid